sklearn/neighbors/tests/test_graph.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
+
+from sklearn.metrics import euclidean_distances
+from sklearn.neighbors import KNeighborsTransformer, RadiusNeighborsTransformer
+
+
+def test_transformer_nonzero():
+    # Test the number of neighbors returned
+    n_neighbors = 5
+    n_samples_fit = 20
+    n_samples_transform = 18
+    n_features = 10
+
+    rng = np.random.RandomState(42)
+    X = rng.randn(n_samples_fit, n_features)
+    X2 = rng.randn(n_samples_transform, n_features)
+    radius = np.percentile(euclidean_distances(X), 10)
+
+    # with n_neighbors
+    for mode in ('distance', 'connectivity'):
```

Comment:
FWIW, I find using tuples for somethings you want to iterate over (and not merely unpack) strange. Tuples express struct of mixed semantics; lists express homogeneous collections.